storops/unity/resource/system.py

```diff
@@ -54,9 +58,40 @@ def get_sp(self, _id=None, name=None, **filters):
         return self._get_unity_rsc(UnityStorageProcessorList, _id=_id,
                                    name=name, **filters)
 
+    def get_iscsi_portal(self, _id=None, name=None, **filters):
+        return self._get_unity_rsc(UnityIscsiPortalList, _id=_id,
+                                   name=name, **filters)
+
+    def get_ethernet_port(self, _id=None, name=None, **filters):
+        return self._get_unity_rsc(UnityEthernetPortList, _id=_id,
+                                   name=name, **filters)
+
+    def get_host(self, _id=None, name=None, **filters):
+        return self._get_unity_rsc(UnityHostList, _id=_id,
+                                   name=name, **filters)
+
+    def get_initiator(self, _id=None, **filters):
+        return self._get_unity_rsc(UnityHostInitiatorList, _id=_id, **filters)
+
     def get_lun(self, _id=None, name=None, **filters):
         return self._get_unity_rsc(UnityLunList, _id=_id, name=name, **filters)
 
+    def create_lun(self, name, pool, size_gb, sp=None, host_access=None,

```

Comment:
It should be a pool method.  Move this method to the pool.  You could leave this method here for simple of use.  And just call `pool.create_lun`.